lib/single-new-eks-awsnative-fargate-observability-construct/index.ts

```diff
@@ -50,7 +56,7 @@ export default class SingleNewEksAWSNativeFargateobservabilityConstruct {
         };
 
         const coreDnsAddOnProps : blueprints.CoreDnsAddOnProps = {
-            version:"v1.10.1-eksbuild.6",
+            versionMap,
```

Comment:
if we set it to auto it will be V1_28 right?
the default mapping is:
KubernetesVersion.V1_28, "v1.10.1-eksbuild.2"
that will be different from v1.10.1-eksbuild.6 selected above, is that ok?

Reply:
Just pass the desired version.